### PR TITLE
lab 5

### PR DESCRIPTION
**Pull request by igorv404**

### CODE REVIEW

File: lib/repositories/establishment_repository.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import 'dart:convert';
+
+import 'package:my_project/models/establishment.dart';
+import 'package:my_project/utils/secure_storage.dart';
+
+class EstablishmentRepository {
```
Comment (DenysDoskochynskyi) [inline lib/repositories/establishment_repository.dart:6]: add abstraction usage)

File: lib/services/user_service.dart [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:my_project/models/login_info.dart';
+import 'package:my_project/models/user.dart';
+
+class UserService {
```
Comment (DenysDoskochynskyi) [inline lib/services/user_service.dart:7]: provide abstraction & extract api url as a const

**Review by DenysDoskochynskyi — CHANGES_REQUESTED** [review]

File: lib/repositories/establishment_repository.dart (hunk 1 above)
Comment (igorv404) [reply lib/repositories/establishment_repository.dart]: Done

File: lib/services/user_service.dart (hunk 2 above)
Comment (igorv404) [reply lib/services/user_service.dart]: Done

**Pull request merged**